### ISSUE TITLE
use mongodb

### ISSUE THREAD

**Issue by premwit555** [prisma/prisma#9003]
Hi Prisma Team! Prisma Migrate just crashed. 

## Command

`db pull`

## Versions
      
| Name        | Version            |
|-------------|--------------------|
| Platform    | darwin             | 
| Node        | v14.17.0           | 
| Prisma CLI  | 2.25.0             | 
| Binary      | c838e79f39885bc8e1611849b1eb28b5bb5bc922| 

## Error
```
Error: [/root/.cargo/git/checkouts/quaint-9f01e008b9a89c14/ed93633/src/single.rs:159:18] not implemented: Supported url schemes: file or sqlite, mysql, postgresql or jdbc:sqlserver.
```

**Comment by janpio** [comment]
Migrations in MongoDB are not supported right now: https://github.com/prisma/prisma/issues/7305

**Comment by matthewmueller** [comment]
Just a quick update because I thought we at least provided a better error message. @premwit555, I'd suggest updating to 2.28.0 or later since we've made a bunch of MongoDB fixes since then.

This crash has been fixed in 2.27.0.
